presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java

```diff
@@ -154,6 +154,8 @@
         TSV,
         CSV_HEADER,
         TSV_HEADER,
+        CSV_WITHOUT_QUOTES,
+        CSV_HEADER_WITHOUT_QUOTES,
```

Comment:
nit. I would group `CSV*` options together.

Reply:
We could shorten the names to make it easier to type. Comparison of some names:
* `CSV_WITHOUT_QUOTES`
* `CSV_NO_QUOTES`
* `CSV_UNQUOTED`